test/testenv/appframework_utils.go

```diff
@@ -366,7 +366,7 @@ func AppFrameWorkVerifications(deployment *Deployment, testenvInstance *TestEnv,
 
 	// Verify apps packages are deleted from the CR pods
 	for _, appSource := range appSource {
-		podDownloadPath := "/init-apps/" + appSource.CrAppSourceVolumeName
+		podDownloadPath := "/operator-staging/appframework/" + appSource.CrAppSourceVolumeName
```

Comment:
you can use the variable name `AppStagingLocOnPod` instead of directly using the directory structure path.